pkg/kubelet/pod_manager.go

```diff
@@ -110,7 +126,37 @@ func (self *basicPodManager) UpdatePods(u PodUpdate, podSyncTypes map[types.UID]
 	}
 }
 
-func updatePods(changed []api.Pod, current []api.Pod) []api.Pod {
+// Need to acquire the lock before calling this function.
+func (self *basicPodManager) SetPods(newPods []api.Pod) {
+	pods := []api.Pod{}
```

Comment:
What does this bring us? That's not going to be the final size for pods....as mirror pods are filtered out from pods.